src/sentry/static/sentry/app/views/requiredAdminActions/setCallsigns.jsx

```diff
@@ -0,0 +1,207 @@
+import React from 'react';
+import {History} from 'react-router';
+import ActionOverlay from '../../components/actionOverlay';
+import OrganizationState from '../../mixins/organizationState';
+import ApiMixin from '../../mixins/apiMixin';
+import {t} from '../../locale';
+import update from 'react-addons-update';
+
+function getProjectInfoForReview(org) {
```

Comment:
This is a pretty gnarly function. I think it could use some comments to help me figure out what's going on.